test/built-ins/Object/prototype/isPrototypeOf/length.js

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2009 the Sputnik authors. All rights reserved.
```

Comment:
```suggestion
// Copyright (C) 2019 Alexey Shvayka. All rights reserved.
```

Reply:
Please see 8ce3790: this test isn't new, but rather merged and refactored to use new harness [`S15.2.4.6_A10.js`](https://github.com/tc39/test262/blob/3aa02c9bbe2f9b1cdb200f356f7ab6ae3693e3ed/test/built-ins/Object/prototype/isPrototypeOf/S15.2.4.6_A10.js), [`S15.2.4.6_A11.js`](https://github.com/tc39/test262/blob/3aa02c9bbe2f9b1cdb200f356f7ab6ae3693e3ed/test/built-ins/Object/prototype/isPrototypeOf/S15.2.4.6_A11.js), [`S15.2.4.6_A8.js`](https://github.com/tc39/test262/blob/3aa02c9bbe2f9b1cdb200f356f7ab6ae3693e3ed/test/built-ins/Object/prototype/isPrototypeOf/S15.2.4.6_A8.js), and [`S15.2.4.6_A9.js`](https://github.com/tc39/test262/blob/3aa02c9bbe2f9b1cdb200f356f7ab6ae3693e3ed/test/built-ins/Object/prototype/isPrototypeOf/S15.2.4.6_A9.js), all authored by Sputnik team.